src/chart.errorBar.js

```diff
@@ -58,54 +95,21 @@ module.exports = function(Chart) {
     },
 
     calculateErrorBarBottom: function(dataset, index, yaxis) {
-      var value = dataset.data[index].y - dataset.data[index].error,
-        ycordinate = yaxis.getPixelForValue(value);
+      var value =
+        dataset.data[index].y - dataset.data[index].error < yaxis.min
```

Comment:
In the future, please avoid ternary operators when the logic and order of operations is not obvious, as in this case. Fine to have code that is uses 1 or 2 more lines if it makes the logic clearer. For example, this could be rewritten as:

```js
var lowestError = dataset.data[index].y - dataset.data[index].error
var value = lowestError < yaxis.min ? yaxis.min : lowestError
```

or even

```js
var lowestError = dataset.data[index].y - dataset.data[index].error
var value = Math.max(lowestError, yaxis.min)
```